BizHawk.Client.EmuHawk/tools/GyroscopeBot/GyroscopeBot.cs

```diff
@@ -168,6 +168,80 @@ private void GyroscopeBot_Load(object sender, EventArgs e)
 
 		#region streetfighter
 
+		/*
+		 * L - High Punch
+X - Middle Punch
+Y - Low Punch
+R - High Kick
+A - Middle Kick
+B - Low Kick
+
+ 
+			 y - y
+			 h - b
+			 n - x
+			 u- a
+			 j - l
+			 m - r
+
+
+			 */
+
+		public uint get_p2_move()
+		{
+			Console.WriteLine("In get_p2_move");
+			if (is_p2_in_move())
+			{
+				/*
+				int _7e5 = _currentDomain.PeekByte(0x0007E5);
+				int _7e6 = _currentDomain.PeekByte(0x0007E6);
+				int _7e7 = _currentDomain.PeekByte(0x0007E7);
+				int _7e8 = _currentDomain.PeekByte(0x0007E8);
+				*/
+				uint val = (uint)_currentDomain.PeekUint(0x0007E5, _bigEndian);
+				return val;
+			}
+			return 0;
+
+		}
+		public bool is_p2_in_move()
+		{
+			if (_currentDomain.PeekByte(0x0007E9) == 0)
+			{
+				return false;
+			}
+			return true;
+		}
+
+		public int p_width_delta()
+		{
+			return _currentDomain.PeekByte(0x0007EB);
+		}
+		public uint get_p1_move()
+		{
+			if (is_p1_in_move())
+			{
+				/*
+				int _7e5 = _currentDomain.PeekByte(0x0007E5);
+				int _7e6 = _currentDomain.PeekByte(0x0007E6);
+				int _7e7 = _currentDomain.PeekByte(0x0007E7);
+				int _7e8 = _currentDomain.PeekByte(0x0007E8);
+				*/
+				uint val = (uint)_currentDomain.PeekUint(0x0005E5, _bigEndian);
```

Comment:
just to confirm, Uint is 4 bytes and why bigEndian?

Reply:
uint is 4 bytes (32 bits) in C#. the _bigEndian check is because the emulator dynamically figures out the endianness of the target emulation engine and sets that property on the tool.